test/bin/build_rpms.sh

```diff
@@ -126,6 +126,14 @@ download_brew_rpms() {
         # Run the download procedure
         bash -x "${SCRIPTDIR}/../../scripts/fetch_tools.sh" brew
         bash -x "${SCRIPTDIR}/manage_brew_rpms.sh" download "4.${MINOR_VERSION}" "${BREW_RPM_SOURCE}"
+
+        out_path="${IMAGEDIR}/released-brew-rpms"
```

Comment:
Do we have to put those in a separate directory?
Nit: if we have to, can we use `brew-rpms-released` name so that they are sorted together in the output?